src/menu.c

```diff
@@ -95,8 +95,18 @@ char *extract_urls(const char *to_match)
  */
 void open_browser(const char *in)
 {
-        // remove prefix and test url
-        char *url = extract_urls(in);
+
+        // remove leading [ linktext ] from URL
```

Comment:
If we're removing the prefix here what's the point of still running it through `extract_urls`?

Reply:
I thought I can catch some weird corner case with this. But after thinking again about it, I've proved myself wrong in this.